packages/vite-plugin-cloudflare/playground/node-compat/worker-als/index.ts

```diff
@@ -0,0 +1,8 @@
+import { AsyncLocalStorage } from "node:async_hooks";
+
+export default {
+	async fetch() {
+		const storage = new AsyncLocalStorage();
+		return storage.run({}, () => new Response("OK!"));
```

Comment:
nit: would it make sense to also test that the `ALS` store actually works? 🤔 

(maybe not since that's a runtime thing anyways...)

Reply:
That is definitely tested elsewhere.

Reply:
ah ok I see, sounds good to me 🙂👍